### DISCUSSION TITLE
删除物化视图后，查看物化视图状态仍存在相关数据

### DISCUSSION THREAD

**Discussion by JohnsonFromTianShan** [apache/doris#17279] [RFC]
Doris-1.2.1中数据表上创建物化视图，在删除之后使用 SHOW ALTER TABLE MATERIALIZED VIEW FROM...语句查看，仍然存在相关物化视图的状态信息，并未得到清除。
![图1](https://user-images.githubusercontent.com/36793478/222061446-9c8b65ec-90cc-47e0-a2d0-e12b9627a6f1.png)
使用 DESC tbl_name ALL语句查看物化视图索引信息，并不显示已删除的物化视图
![图2](https://user-images.githubusercontent.com/36793478/222061700-51057f52-af15-438d-9d19-3c91d1dca70d.png)

**Comment by LemonLiTree** [comment]
SHOW ALTER TABLE MATERIALIZED VIEW 是显示CREATE MATERIALIZED VIEW 语句提交的创建物化视图作业的执行情况。
即使已经删除，也会显示的吧

**Reply by JohnsonFromTianShan** [reply]
哦哦！官方文档也的确是如此表述的，是我理解不深刻造成的。
不过，依我使用数据库的习惯看：既然物化视图都不存在了，留下作业执行记录似乎已无必要。皮之不存，毛成“僵尸”！
![官方文档说明](https://user-images.githubusercontent.com/36793478/223968082-f3e5d804-09a4-4276-ad03-e3433e82846c.png)
